src/games/strategy/engine/chat/PlayerChatRenderer.java

```diff
@@ -47,13 +46,12 @@ public Component getListCellRendererComponent(final JList<?> list, final Object
       final Set<String> players = m_playerMap.get(value.toString());
       if (players != null && !players.isEmpty()) {
         sb.append(" (");
-        final Iterator<String> iter = players.iterator();
-        while (iter.hasNext()) {
-          sb.append(iter.next());
-          if (iter.hasNext()) {
-            sb.append(", ");
-          }
+        for(String player : players) {
```

Comment:
Use Guava Joiner instead, it'll be something like:
`Joiner.on(",").join(players)`